### PR TITLE
Da/unsafe keyshare upload

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/common/stores/applications/SsvWeb/Validator.store.ts [hunk 1]
```diff
@@ -655,6 +655,156 @@ class ValidatorStore extends BaseStore {
...
+    });
+  }
+
+  async createKeySharePayloadUnsafe(update: boolean = false): Promise<Map<string, any> | false> {
+    update;
+    return new Promise(async (resolve) => {
+      const ssvStore: SsvStore = this.getStore('SSV');
+      const walletStore: WalletStore = this.getStore('Wallet');
+      const clusterStore: ClusterStore = this.getStore('Cluster');
+      let totalCost = 4.5;
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Validator.store.ts:750]: done

File: src/app/common/stores/applications/SsvWeb/Validator.store.ts [hunk 2]
```diff
@@ -655,6 +655,156 @@ class ValidatorStore extends BaseStore {
...
+            }
+            console.debug('Contract Error', error);
+            resolve(true);
+          });
+    });
+  }
+
+  async createKeySharePayloadUnsafe(update: boolean = false): Promise<Map<string, any> | false> {
+    update;
+    return new Promise(async (resolve) => {
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Validator.store.ts:746]: done

File: src/app/common/stores/applications/SsvWeb/Validator.store.ts [hunk 3]
```diff
@@ -655,6 +655,154 @@ class ValidatorStore extends BaseStore {
...
+      CATCH_ERROR_RESPONSE,
+      VALIDATOR_EXIST_RESPONSE,
+      VALIDATOR_PUBLIC_KEY_ERROR,
+    } = translations.VALIDATOR.KEYSHARE_RESPONSE;
+    try {
+      const fileJson = await this.keyShareFile?.text();
+      const operatorStore: OperatorStore = this.getStore('Operator');
+      const walletStore: WalletStore = this.getStore('Wallet');
+      await accountStore.getOwnerNonce(walletStore.accountAddress);
+      const { ownerNonce } = accountStore;
```
Comment (andrew-blox) [inline src/app/common/stores/applications/SsvWeb/Validator.store.ts:780]: Please add a console.log with ownerNonce, doesnt have to be at this point but somewhere during the registration flow i would really like to see such a log

`Current registration nonce: ${ownerNonce}`

**Review by andrew-blox — CHANGES_REQUESTED** [review]
Aside from the console.log I want to make sure that we will not be blocked in IL for these transactions in mainnet when we try to register, @Tomblox1 @axelrod-blox lets make sure on this

File: src/app/common/stores/applications/SsvWeb/Validator.store.ts (hunk 3 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Validator.store.ts:780]: added

Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Validator.store.ts:780]: done

**Review by andrew-blox — APPROVED** [review]

**Pull request merged**